src/teleop/gui/src/components/ArmControls.vue

```diff
@@ -29,7 +29,7 @@ export default {
     };
   },
 
-  beforeUnmount: function () {
+  beforeDestroy: function () {
```

Comment:
why was this changed to beforeDestroy? It seems like it [should ](https://stackoverflow.com/questions/66305505/strange-vue-warn-beforedestroy-has-been-renamed-to-beforeunmount)be called beforeUnmount.

Reply:
Turns out beforeUnmount is a Vue3 hook, but we are on vue 2 currently. That is why I also switched the eslint preset. Would be good to switch to vue 3 sometime but probably not this year.